accounts-db/src/read_only_accounts_cache.rs

```diff
@@ -137,7 +137,7 @@ impl ReadOnlyAccountsCache {
         }
     }
 
-    pub(crate) fn load(&self, pubkey: Pubkey, slot: Slot) -> Option<AccountSharedData> {
+    pub fn load(&self, pubkey: Pubkey, slot: Slot) -> Option<AccountSharedData> {
```

Comment:
nit: here and in store, make these public only under the dev-context-only-utils
feature. When reading code there's value in knowing whether something is
actually used by the validator or if it's being exported for tests/benches.

Reply:
Honest dumb question - how?

If this was a regular function, I could simply do something like:

```rust
#[cfg(feature = "dev-context-only-utils")]
pub use load;
```

But for the methods, the only way I can imagine is:

```rust
impl ReadOnlyAccountsCache {
    #[cfg(feature = "dev-context-only-utils")]
    pub fn load(&self, pubkey: Pubkey, slot: Slot) -> Option<AccountSharedData> {
        self._load(pubkey,slot)
    }

    #[cfg(not(feature = "dev-context-only-utils"))]
    fn load(&self, pubkey: Pubkey, slot: Slot) -> Option<AccountSharedData> {
        self._load(pubkey, slot)
    }

    fn _load(&self, pubkey: Pubkey, slot: Slot) -> Option<AccountSharedData> {
        // do stuff
    }
}
```

Is there any other less dumb trick I'm missing?

Reply:
Thanks a lot, I didn't know about the `qualifier_attr` crate, neat